### PR TITLE
Add !top command and fix accuracy formatting

### PR DESCRIPTION
**Pull request by kanaarima**
This solves #6.

### CODE REVIEW

File: app/commands/top.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    if len(context.args):
+        if context.args[0] in modes:
+            mode = modes[context.args[0]]
+        else:
+            await context.message.reply(
+                f"Wrong mode! Available modes: {', '.join(modes.keys())}"
+            )
+            return
+
+    user_scores = scores.fetch_top_scores(user_id=user.id, mode=mode)[:10]
```
Comment (Lekuruu) [inline app/commands/top.py:27]: I would actually add a `limit` parameter here. Let me implement that really quick...

File: app/commands/top.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        mods = Mods(score.mods).short
+        beatmap_title = (
+            f"{score.beatmap.beatmapset.title} [{score.beatmap.version}] +{mods}"
+        )
+        str_builder += f"{position}. {beatmap_title}\n"
+        str_builder += f"   {rank} {max_combo}/{score.beatmap.max_combo} {accuracy*100:.2f}% [{n300}/{n100}/{n50}/{nmiss}] {pp:.2f}pp\n"
+        position += 1
+
+    embed = Embed(
+        title=f"Top play for {user.name}",
```
Comment (Lekuruu) [inline app/commands/top.py:53]: "Top plays" instead of "Top play"?

Comment (kanaarima) [reply app/commands/top.py:53]: whoops

File: app/commands/top.py (hunk 1 above)
Comment (Lekuruu) [reply app/commands/top.py:27]: @kanaarima can you sync the fork and add `limit=10`?

**Review by Lekuruu — APPROVED** [review]
lgtm

**Comment by Lekuruu** [comment]
We should definitely remove that peppy image someday lmao

**Comment by kanaarima** [comment]
> We should definitely remove that peppy image someday lmao

I think its great

**Pull request merged**